src/Checkbox/CheckboxComponent.jsx

```diff
@@ -13,10 +12,12 @@ export default observer(() => {
         id="example-checkbox"
         name="foo"
         onChange={(event) => {
-          checkboxPresenter.setIsChecked(event.target.checked)
+          dispatch(
+            {type: 'SET_IS_CHECKED', payload: event.target.checked}
+          );
```

Comment:
This is how you mutate state. 

It's a simplified version of the Redux API, though it's not Redux.

Reply:
We could export a function from `reducers()` that does this same thing, but it'd just be a pass-through:

```js
function setIsChecked(dispatch, isChecked) {
  dispatch(
    {type: 'SET_IS_CHECKED', payload: isChecked}
  );
```